services/platform/platform.go

```diff
@@ -53,14 +53,21 @@ type supervisordService interface {
 	UpdateConfiguration(settings *models.Settings, ssoDetails *models.PerconaSSODetails) error
 }
 
+// type checksService is a subset of methods of checks.Service used by this package.
+// We use it instead of real type for testing and to avoid dependency cycle.
+type checksService interface {
+	CollectChecks(ctx context.Context)
+}
```

Comment:
Usually we keep interfaces in separate file `deps.go` in a package folder. Let's move there this one and other two)